### PR TITLE
PADV-1927 feat: add page size param to services

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-1927

### Description
add page size param to services

### Changes made
Add page size param to services, to get max 200 records 
Add spinner loaders where is missing 
Update test

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1980/institution-portal

### CODE REVIEW

File: src/features/Dashboard/InstructorAssignSection/index.jsx [hunk 1]
```diff
@@ -47,13 +50,22 @@ const InstructorAssignSection = () => {
...
+        {isLoadingClasses && (
+          <div className="w-100 h-100 d-flex justify-content-center align-items-center">
+            <Spinner
+              animation="border"
+              className="mie-3"
+              screenReaderText="loading"
+            />
+          </div>
+        )}
+        {!isLoadingClasses && classCards.map(classInfo => <ClassCard data={classInfo} key={classInfo?.classId} />)}
```
Comment (01001110J) [inline src/features/Dashboard/InstructorAssignSection/index.jsx:62]: Great addition, but I would like to suggest that changes outside the scope of the issue should go into a separate PR. This way, changes won't get mixed, and it will make the review process easier by avoiding unrelated updates to the issue's scope. Additionally, if a revert is needed, it would only affect a specific change rather than multiple changes at once.

**Review by 01001110J — COMMENTED** [review]
This change won't be added to `events`?

**Comment by AuraAlba** [comment]
> This change won't be added to `events`?

Same as instructors portal, we don't need pages in this component and we already implement a function to call all pages, so we don't change to avoid performance issues

**Review by 01001110J — APPROVED** [review]
